src/pspm_options.m

```diff
@@ -404,7 +403,7 @@
         return
       end
     end
-    if options.change_data == 0 && ~isfield(options, 'missing_epochs_filename')
+    if options.change_data == 0 && ~isempty(options.missing_epochs_filename)
```

Comment:
Should this be isempty rather than ~isempty?

Reply:
Apologies, this is my fault here. I have resolved it now by correcting this line of code and adding the corresponding test functions.